src/credential/sd_jwt_vc.rs

```diff
@@ -1,7 +1,123 @@
+use std::sync::Arc;
+
 use ssi::claims::{
     jwt::{AnyClaims, JWTClaims},
     sd_jwt::{RevealedSdJwt, SdJwtBuf},
 };
+use uuid::Uuid;
+
+use super::{jwt_vc::JwtVc, Credential, ParsedCredential, ParsedCredentialInner};
+
+#[derive(Debug, uniffi::Object)]
+pub struct SdJwtVc {
+    raw: Vec<u8>,
```

Comment:
NOTE: Using the raw payload here instead of parsed fields to re-use the `JwtVc` type.

We may wish to capture these same fields as `JwtVc` or provide accessor methods.